### DISCUSSION TITLE
Interpretability for 3D Unet segmentation

### DISCUSSION THREAD

**Discussion by HripsimeS** [Project-MONAI/tutorials#731] [Q&A]
Dear MONAI team, @rijobro @Nic-Ma 

I have seen your great tutorials of interpretability study for classification models:  
https://github.com/Project-MONAI/tutorials/tree/main/modules/interpretability

Do you have also some tutorials of interpretability study for 3D Unet segmentation? 
I am not sure it will work if I just change the classification model to segmentation model in tutorials.

Thanks in advance!

**Comment by wyli** [comment]
this is answered https://github.com/Project-MONAI/MONAI/issues/3032 https://github.com/Project-MONAI/MONAI/discussions/3005
